doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -99,28 +99,30 @@ These specifications cover the [full lifecycle of a container](https://github.co
 
 > Editors note: The following chapter lists a set of CNI plugins conformant with the Reference Architecture. In future releases the list of CNI plugins should be refined in a way that there is only component selected for each functionality. 
 
-The used CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm)
-as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).<br>
-
-The following table contains a comparision of relevant features and requirements in Multus and DANM.
-
-| Requirement | Support in Multus | Support in DANM |
-|-------------|-------------------|-----------------|
-| `req.inf.ntw.01` | Supported | Supported |
-| `req.inf.ntw.02` | Supported via an other CNI plugin | Supported via an other CNI plugin |
-| `req.inf.ntw.03` | Supported via an other CNI plugin | Supported |
-| `req.inf.ntw.04` | Supported via an other CNI plugin | Supported via an other CNI plugin |
-| `req.inf.ntw.06` | Supported | Supported |
-| `req.inf.ntw.07` | Supported | Supported |
-| `req.inf.ntw.08` | Supported | Supported |
-| `req.inf.ntw.09` | Supported via LCM tools |  Supported via LCM tools |
-| `req.inf.ntw.10` | Not supported | Suported |
-| `req.inf.ntw.11` | Not supported | Partially supported |
-| `req.inf.ntw.14` | Supported via an other CNI plugin | Supported via an other CNI plugin |
-| `req.inf.ntw.15` | Not relevant | Not relevant |
-| `req.inf.ntw.16` | Supported | Supported |
-| Cluster wide IP address management | Not suported | Supported |
-| Service based dicovery of all provisioned interfaces | Not supported | Supported |
+The used CNI multiplexer/metapulgin may be [DANM](https://github.com/nokia/danm) as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).<br>
+
+The NSX Container Plugin ([NCP](https://kubernetes.io/docs/concepts/cluster-administration/addons0/)) is a CNCF-compliant container plugin that integrates with NSX-T Data Center to build Kubernetes networking and security objects.  NSX-T Container Plugin natively supports Containers in VM deployment model with superior security/isolation while meeting the data plane performance requirements. The NCP can also serve as a metaplugin/CNI multiplexer that is capable of attaching multiple interfaces to Kubernetes Pods to provide multiple NIC container support.
```

Comment:
- The link you provided gives a 404
- What does "CNCF Compliant" mean in this context? Isn't CNI compliant what you wanted to write?
- Metaplugin means, that the CNI plugin can "control" other CNI plugins. Is NCP capable of this?
- Can you please provide a link to NSX Container Plugin?